CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 <br>
 
+## [1.3.0] - 07/08/2023
+
+### Added
+- [DEL-31] Added toggle mechanism for displaying a triangle's circum-circle when clicking inside its body
+
+<br>
+
 ## [1.2.3] - 04/08/2023
 
```

Comment:
Thanks for the minor version bump. Missing the compare link at bottom of page e.g
`[1.3.0]: https://github.com/JRSmiffy/delaunay/compare/1.2.3...1.3.0` <br>
Which can be added even at this stage (when 1.3.0 doesn't exist yet)